triton_backend/all_models/tests/test_llmapi_python_backend.py

```
@@ -216,6 +217,109 @@ def test_convert_request_input_to_dict():
+                                                      lora_int_id=42,
+                                                      lora_path=adapter_dir,
+                                                      lora_ckpt_source="nemo")
+
+        # Invalid lora_ckpt_source -> raises (must be hf or nemo).
+        request = make_mock_triton_request({
+            "lora_id": [42],
+            "lora_name": [b"a"],
+            "lora_path": [adapter_dir.encode("utf-8")],
+            "lora_ckpt_source": [b"bogus"],
+        })
+        with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)